src/amo/components/AddonBadges/styles.scss

```diff
@@ -12,7 +12,6 @@
 }
 
 .AddonBadges .Badge {
-  @include margin-start(6px);
```

Comment:
@tofumatt we need to remove this from #4707 as  `theme image` and `badge` are on different rows on mobile.

Reply:
@tofumatt on desktop its handled in line https://github.com/mozilla/addons-frontend/blob/master/src/amo/components/AddonBadges/styles.scss#L19
On screen less than `large` we don't need this, where badges are not on `right-top` but below title and description.

Themes are not impacted by this patch 
<img width="654" alt="screen shot 2018-04-04 at 8 33 21 am" src="https://user-images.githubusercontent.com/5318732/38286428-fa43fedc-37e2-11e8-8d7e-4d780b4a2f96.png">
<img width="312" alt="screen shot 2018-04-04 at 8 33 06 am" src="https://user-images.githubusercontent.com/5318732/38286429-fa742698-37e2-11e8-858d-920eb7ca1414.png">
<img width="870" alt="screen shot 2018-04-04 at 8 32 54 am" src="https://user-images.githubusercontent.com/5318732/38286430-faa3489c-37e2-11e8-8e88-4fabfbcf2f07.png">

Reply:
Ah, okay, but is that featured theme badge on the large size off a bit? It seems like there's a lot of padding on it, pushing it to the left a bit more than expected.

Maybe that's just the regular padding.